### PR TITLE
feat: added ability to point to kms in different account

### PR DESCRIPTION
**Pull request by Ak-sky**
### Description

Added ability to point to kms in different account.

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [x] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -247,6 +247,12 @@ variable "key_protect_use_private_endpoint" {
   default     = true
 }
 
+variable "kms_cross_account_id" {
+  type        = string
+  description = "Id of the account that owns the KMS instance to encrypt the cluster"
+  default     = null
+}
```
Comment (ocofaigh) [inline variables.tf:254]: suggest just calling it `kms_account_id` - mention that its only required if the KMS instance is in another account

Comment (Ak-sky) [reply variables.tf:254]: Done.

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**